# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1472668653 14400
#      Wed Aug 31 14:37:33 2016 -0400
# Node ID bc38c1ff6698abbc1ef33e6b7c24f15e5f4075b2
# Parent  fda8d91b2e716f53844827f1aeb1241c6edfea31
fix definition of F77_RETURN for gfortran

* f77-fcn.h (F77_RETURN) [F77_USES_GFORTRAN_CALLING_CONVENTION]:
Fix definition to match F77_RET_T == void.

diff --git a/liboctave/cruft/misc/f77-fcn.h b/liboctave/cruft/misc/f77-fcn.h
--- a/liboctave/cruft/misc/f77-fcn.h
+++ b/liboctave/cruft/misc/f77-fcn.h
@@ -277,17 +277,17 @@ octave_make_cray_const_ftn_ch_dsc (const
 #define F77_CHAR_ARG_DEF(s, len) char *s
 #define F77_CONST_CHAR_ARG_DEF(s, len) const char *s
 #define F77_CHAR_ARG_LEN_DEF(len) , F77_CHAR_ARG_LEN_TYPE len
 #define F77_CHAR_ARG_USE(s) s
 #define F77_CHAR_ARG_LEN_USE(s, len) len
 
 #define F77_RET_T void
 
-#define F77_RETURN(retval) return retval;
+#define F77_RETURN(retval) return;
 #if defined (HAVE_OCTAVE_NORETURN_ATTR)
 #  define F77_NORETURN(retval)
 #else
 #  define F77_NORETURN(retval) return retval;
 #endif
 
 #elif defined (F77_USES_F2C_CALLING_CONVENTION)
 
